tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/FieldSymbolTest.java

```diff
@@ -120,6 +121,20 @@ public void testClassFields(int line, int col, String fieldName, TypeDescKind ty
         };
     }
 
+    @Test
+    public void testRecordFieldSymbolPos() {
+        Symbol symbol = getSymbol(51, 14);
+        assertEquals(symbol.kind(), SymbolKind.RECORD_FIELD);
+
+        RecordFieldSymbol fieldSymbol = (RecordFieldSymbol) symbol;
+        Location symbolPos = fieldSymbol.getLocation().get();
```

Comment:
```suggestion
        assertTrue(fieldSymbol.getLocation().isPresent());
        Location symbolPos = fieldSymbol.getLocation().get();
```

Missing a presence check here.

Reply:
Addressed with 0ab4473